config/initializers/statsd.rb

```diff
@@ -0,0 +1,5 @@
+require 'datadog/statsd'
+STATSD = Datadog::Statsd.new(
+  ENV['DD_AGENT_PORT_8126_TCP_ADDR'],
+  (ENV['DD_AGENT_STATSD_PORT'] || 8125).to_i,
+  {namespace: ENV["DD_AGENT_NAMESPACE"]})
```

Comment:
I like this - when we write instructions we should include this in the boilerplate.  We should add it to fixops too.  I did `sf.` for the logs (instead of `heroku.logs.`, so unless you think it should be something different I'm going to set it to that.